src/main/kotlin/no/nav/dagpenger/regel/api/db/PostgresSubsumsjonStore.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.dagpenger.regel.api.db
+
+import kotliquery.queryOf
+import kotliquery.sessionOf
+import javax.sql.DataSource
+
+class PostgresSubsumsjonStore(private val dataSource: DataSource) : SubsumsjonStore {
+
+    override fun isHealthy() = sessionOf(dataSource).run(queryOf("SELECT 1").asExecute)
+
+    override fun insert(subsumsjonsId: String, json: String) {
+        sessionOf(dataSource).run(queryOf(
+                """
+            INSERT INTO subsumsjon VALUES (?, (to_json(?::json)))
+            ON CONFLICT(ulid) DO UPDATE SET data = (to_json(?::json))
```

Comment:
not sure about the  on conflict, the subsumsjonsId is unique, but may happen on retries etc from Kafka consuming,